src/Pages/Login/Login.js

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import CommonInput from '../../Components/CommonInput/CommonInput';
+import './Login.scss';
+
+class Login extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      phoneValue: '',
+      passwordValue: '',
+      isLogin: false,
+      isWarning: [false, false],
+    };
+  }
+
+  checkInputLenght = () => {
+    const { phoneValue, passwordValue } = this.state;
+    this.setState({
+      isLogin: phoneValue.length > 0 && passwordValue.length > 0 ? true : false,
+    });
+  };
+
+  handleInput = e => {
+    const { name, value } = e.target;
+    this.setState(
+      {
+        [name]: value,
+      },
+      () => {
+        this.checkInputLenght();
+      }
+    );
+  };
```

Comment:
```suggestion
 handleInput = e => {
    const { name, value } = e.target;
    this.setState({
      [name]: value,
    });
  };
```
로그인 버튼 활성화 시키는 부분을 삭제하고 아래 변수를 활용해서 사용하도록 할게요!!
state를 사용하지 않아도 변수만 활용해서 할 수 있습니다!

Reply:
수정했습니다